File: src/main/java/io/junseok/mealmate/domain/membercategory/repository/MemberCategoryRepository.java
package io.junseok.mealmate.domain.membercategory.repository;

import io.junseok.mealmate.domain.membercategory.entity.MemberCategory;
import org.springframework.data.jpa.repository.JpaRepository;

public interface MemberCategoryRepository extends JpaRepository<MemberCategory,Long> {

}

File: src/main/java/io/junseok/mealmate/domain/membercategory/repository/MemberCategoryRepository.kt
package io.junseok.mealmate.domain.membercategory.repository

import io.junseok.mealmate.domain.member.entity.Member
import io.junseok.mealmate.domain.membercategory.entity.MemberCategory
import org.springframework.data.jpa.repository.JpaRepository

interface MemberCategoryRepository : JpaRepository<MemberCategory, Long> {
    fun findAllByMember(member: Member): List<MemberCategory>
}
